extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -256,6 +259,13 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     if (this.ampAnalyticsConfig_) {
       insertAnalyticsElement(this.element, this.ampAnalyticsConfig_, true);
     }
+    const frame = this.element.querySelector('iframe');
+    if (frame) {
```

Comment:
Rather than attempting to find the iframe, why don't we make it a public field within A4AElement that you can access.  You can then add a dev().assert(!!this.iframe)